src/pages/EditorPage.js

```diff
@@ -565,37 +741,20 @@ export default function EditorPage(props) {
                   />
                 </div>
                 <div className="mb-3 d-flex gap-2 flex-wrap">
-                  <button
-                    className="btn btn-success"
-                    onClick={() => {
-                      setRenderCode(code);
-                      if (layout === Layout.Tabs) {
-                        setTab(Tab.Widget);
-                      }
-                    }}
-                  >
-                    Render preview
-                  </button>
-                  {!path?.unnamed && commitButton}
-                  <button
-                    className={`btn ${
-                      path?.unnamed ? "btn-primary" : "btn-secondary"
-                    }`}
-                    onClick={() => {
-                      setShowRenameModal(true);
-                    }}
-                  >
-                    Rename {path?.type}
-                  </button>
-                  {path && accountId && (
-                    <a
-                      className="btn btn-outline-primary"
-                      href={`#/${widgetPath}`}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      Open Component in a new tab
-                    </a>
+                  {isDraft ? (
+                    <>
+                      {saveDraftButton}
+                      {saveLocallyButton}
+                      {publishDraftAsMainButton}
```

Comment:
@marcinbodnar @milenakrawczyk should we force users to save an in-progress component on draft as a chain before offering to `Publish draft as main version`?

Reply:
I think it's a good idea. This would solve a problem of divergent draft versions. If we force users to save locally, a locally saved version is always newer or equal to the chain version. It is very clean. What do you think @marcinbodnar ?

Reply:
I think `Publish draft as main version` should be available/visible only for a draft component

Reply:
Maybe we may consider forcing to save locally only. `Publishing draft as main version` nullifies the draft version on chain in the current implementation so either we force to save locally or somehow change the current implementation.